pykwasm/src/pykwasm/kdist/plugin.py

```diff
@@ -55,6 +55,48 @@ def deps(self) -> tuple[str]:
         return ('wasm-semantics.source',)
 
 
+def ulm_wasm_args(src_dir: Path, ulm_test: bool = False) -> dict[str, Any]:
+    build_dir = Path(__file__).parent.parent.parent.parent.parent / 'build'
+    print(build_dir)
```

Comment:
I'd remove this print statement unless we have a separate flag for debugging the build script to gate it behind.

Reply:
sorry. I had forgotten about it.